README.md

```diff
@@ -8,31 +12,52 @@ punctuation substitutions, etc.), and it is safe for all utf-8
 (unicode) input.
 
 HTML output is currently supported, along with Smartypants
-extensions. An experimental LaTeX output engine is also included.
+extensions.
 
 It started as a translation from C of [Sundown][3].
 
 
 Installation
 ------------
 
-Blackfriday is compatible with Go 1. If you are using an older
-release of Go, consider using v1.1 of blackfriday, which was based
-on the last stable release of Go prior to Go 1. You can find it as a
-tagged commit on github.
-
-With Go 1 and git installed:
+Blackfriday is compatible with any modern Go release. With Go 1.7 and git
+installed:
 
-    go get github.com/russross/blackfriday
+    go get gopkg.in/russross/blackfriday.v2
 
 will download, compile, and install the package into your `$GOPATH`
 directory hierarchy. Alternatively, you can achieve the same if you
```

Comment:
This "Alternatively" section should probably be removed. It's easier to tell people to `go get -u` the import path, they can figure out the rest by now.